.rubocop.yml

```diff
@@ -1,21 +1,41 @@
+inherit_from: .rubocop_todo.yml
+
 require:
   - rubocop-rails
+  - rubocop-rspec
+  - rubocop-performance
 
 AllCops:
-  TargetRubyVersion: 2.5
-  NewCops: disable
+  TargetRubyVersion: 3.0
+  NewCops: enable
+  DisplayCopNames: true
+  DisplayStyleGuide: true
+  ExtraDetails: true
+  UseCache: true
+  CacheRootDirectory: tmp
   Exclude:
-    - 'spec/**/*'
-    - 'db/**/*'
-    - 'app/views/**/*'
-    - 'config/**/*'
-    - 'bin/*'
-    - 'Rakefile'
-    - 'node_modules/**/*'
-    - 'Vagrantfile'
-    - 'vendor/**/*'
-    - 'lib/json_ld/*'
-    - 'lib/templates/**/*'
+    - bin/*
+    - build/**/*
+    - chart/**/*
+    - config.ru
+    - config/boot.rb
+    - config/environment.rb
+    - config/puma.rb
+    - coverage/**/*
+    - db/schema.rb
+    - dist/**/*
+    - log/**/*
+    - node_modules/**/*
+    - public/**/*
+    - Rakefile
+    - redis/**/*
+    - streaming/**/*
+    - templates/**/*
+    - tmp/**/*
+    - Vagrantfile
+    - vendor/**/*
+    # Syntax error with ruby erb template file
+    - lib/templates/rails/post_deployment_migration/migration.rb
```

Comment:
This change will be gone with what I will do with the changes I will need to make to this PR.

With the comment I made on your PR yesterday, I realised that doing this with the todo exclude file I made is actually wrong, because the CI only seems to lint changed code.

Because if that, the idea to have RuboCop passing on the whole project is actually not needed and wanted.
So remooving the todo thing, I'll have to rewind some rule changes I made in .rubocop.yml as well. They'd be to much of an interference from me, if there is no generated excludes file.

So thats why I had to make this PR draft again.

I will bring this back with much softer aporoach according rule changes.

I will still add and include the specs though.
And will explain the rule changes made more detailed, so they can be adressed and accepted or denied in the PR review.

I actually only started this rubo cop changes, because I wanted to have RuboCop autocorrect in the specs I write in another PR 😆
It seems to be a more complicated thing though, as it turns out. 😉